crates/solidity/Cargo.toml

```diff
@@ -40,6 +40,17 @@ inkwell = { workspace = true }
 revive-common = { path = "../common" }
 revive-llvm-context = { path = "../llvm-context" }
 
-
 [target.'cfg(target_env = "musl")'.dependencies]
 mimalloc = { version = "*", default-features = false }
+
+[target.'cfg(target_os = "emscripten")'.dependencies]
+libc = { workspace = true }
+inkwell = { workspace = true, features = ["target-riscv", "llvm18-0-no-llvm-linking"]}
+
+[features]
+default = ["parallel"]
+parallel = ["rayon"]
+
+#TODO: Use pthreads -C target-feature=+atomics,+bulk-memory -Clink-arg=-pthread in compilation and enable `parallel` feature
```

Comment:
Do you plan to resolve this TODO in this PR? If not and it is blocked by some larger issue can you please point out why?

Reply:
I don't plan to do it now. I spent a couple of hours on it and still had issues where some libraries were compiled without the -pthread flag.